docs/trace_collection.md

```diff
@@ -221,6 +221,23 @@ val okHttpClient =  OkHttpClient.Builder()
 
 This creates a span around each request processed by the OkHttpClient, with all the relevant information automatically filled (url, method, status code, error), and propagates the tracing information to your backend to get a unified trace within Datadog
 
+Because the way the OkHttp Request is executed (using a Thread pool), we are not able to automatically detect the active parent span and add it to the Request span. You can still manually provide a parent Span in the OkHttpRequest Builder as follows:
```

Comment:
> we are not able to automatically detect the active parent span 
This is misleading, and slightly untrue. If there was an active span on the okhttp thread, we definitely could detect it, and use it as parent. I'd phrase at as : 

> we are not able to link the request span with the span that triggered the request.

Also it's `OkHttp Request.Builder` instead of `OkHttpRequest Builder`